src/macros.rs

```diff
@@ -329,51 +325,54 @@ macro_rules! service {
             rpc $fn_name( $( $arg : $in_ ),* ) -> $out | $error;
         }
     };
-    // Pattern for when all return types have been expanded
+// Pattern for when all return types have been expanded
```

Comment:
We can leave them as bad if you want. There's lots of merit to being able to rustfmt everything. Do you know if there's an issue open for this?